tests/utils.test.js

```diff
@@ -1,4 +1,4 @@
-const { bline, drawCircle, drawDisc, maskBlack } = require('../js/utils.js');
```

Comment:
the file contents should be kept as it was before

don't remove anything

only add necessary tests